### DISCUSSION TITLE
Adjust artist name

### DISCUSSION THREAD

**Discussion by catap** [beetbox/beets#4059] [Q&A]
Let take as example Rudy Linka: https://musicbrainz.org/artist/a98a3b1a-b219-4c27-bc09-bb1447003fad

He has artist name for some albums with a lot of members which forces beets to use all this huge name inside path.

Or https://musicbrainz.org/release/36629049-127c-4c3f-8fbe-ed269611f631 which some not latin symbols which someone may would like to adjust. 

Is it possible to keep (somewhere) preferred artist name?

**Comment by sampsyo** [comment]
This `languages` setting can help identify the MusicBrainz alias to use:
https://beets.readthedocs.io/en/stable/reference/config.html#languages

**Reply by edgars-supe** [reply]
I think it would be quite useful to add a flag to `import` with which to set/override the `languages` config. For example, ordinarily I don't use the config, but when I'm importing classical or world music, I'd rather have the artist's name in English.

**Reply by sampsyo** [reply]
FWIW, you can "fake" such a command-line flag using the `-c` flag… that is, `-c extra_config.yaml` where that file contains just the option you want to override will let you modify your configuration for a single command run.

**Reply by catap** [reply]
@edgars-supe I concern more about the case when you run `mbsync` and after that it tries to rename some artists or albums.

Sometime I think that it is ok, sometime I dissagree and idea to keep musicbrainz on the way as I prefer to sorting my music is wrong.

The right way is keep local mapping. For example I'd love to keep all albums of `Rudy Linka` inside `Rudy Linka` artist, not a list with whom he recorded it.

**Comment by catap** [comment]
After a thinking I bit I'd like to suggest to plugin similar with `edit` (or extension for edit?) which allow to mark some field as "not updated" from musicbrainz.

**Comment by trapd00r** [comment]
How about this: https://github.com/beetbox/beets/discussions/4012

**Reply by catap** [reply]
Can work but quite complicated.
